examples/asrc_demo/bsp_config/XK_VOICE_L71/XK_VOICE_L71.xn

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Network xmlns="http://www.xmos.com"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://www.xmos.com http://www.xmos.com">
+  <Declarations>
+    <Declaration>tileref tile[2]</Declaration>
+    <Declaration>tileref usb_tile</Declaration>
+  </Declarations>
+
+  <Packages>
+    <Package id="0" Type="XS3-UnA-1024-QF60A">
+      <Nodes>
+        <!-- Note that this clock setting is overridden by the app by writing directly to the PLL -->
+        <Node Id="0" InPackageId="0" Type="XS3-L16A-1024" SystemFrequency="600MHz" Oscillator="24MHz" referencefrequency="100MHz">
+          <Boot>
+            <Source Location="bootFlash"/>
+          </Boot>
+<!--           <Extmem sizeMbit="1024" Frequency="175MHz">
+            <Padctrl clk="0x0" cke="0x0" cs_n="0x0" we_n="0x0" cas_n="0x0" ras_n="0x0" addr="0x0" ba="0x0" dq="0x0" dqs="0x0" dm="0x0"/>
+            <Lpddr lmr_opcode="0x0" emr_opcode="0x0"/>
+          </Extmem> -->
+          <Tile Number="0" Reference="tile[0]">
+            <!-- QSPI ports -->
+            <Port Location="XS1_PORT_1B"  Name="PORT_SQI_CS_0"/>
+            <Port Location="XS1_PORT_1C"  Name="PORT_SQI_SCLK_0"/>
+            <Port Location="XS1_PORT_4B"  Name="PORT_SQI_SIO_0"/>
+
+            <!-- SPI ports -->
+            <Port Location="XS1_PORT_1A"  Name="PORT_SSB"/>
+            <Port Location="XS1_PORT_1C"  Name="PORT_SQI_SCLK_0"/>
+            <Port Location="XS1_PORT_1D"  Name="PORT_SPI_MOSI"/>
+            <Port Location="XS1_PORT_1P"  Name="PORT_SPI_MISO"/>
+
+            <!-- I2C ports -->
+            <Port Location="XS1_PORT_1N"  Name="PORT_I2C_SCL"/>
+            <Port Location="XS1_PORT_1O"  Name="PORT_I2C_SDA"/>
+
+            <!-- GPIO ports -->
+            <Port Location="XS1_PORT_8C"  Name="PORT_GPO"/>
+            <Port Location="XS1_PORT_8D"  Name="PORT_GPI"/>
+
+            <!-- Used for keeping XUA happy only -->
+            <Port Location="XS1_PORT_1G"  Name="PORT_NOT_IN_PACKAGE_0"/>
+
+          </Tile>
+          <Tile Number="1" Reference="tile[1]">
+
+            <!-- MIC related ports -->
+            <Port Location="XS1_PORT_1G"  Name="PORT_PDM_CLK"/>
+            <Port Location="XS1_PORT_1F"  Name="PORT_PDM_DATA"/>
+
+            <!-- Audio ports -->
+            <Port Location="XS1_PORT_1D"  Name="PORT_MCLK_IN_OUT"/>
+            <Port Location="XS1_PORT_1C"  Name="PORT_I2S_BCLK"/>
+            <Port Location="XS1_PORT_1B"  Name="PORT_I2S_LRCLK"/>
+            <Port Location="XS1_PORT_1A"  Name="I2S_MIC_DATA"/>
+            <Port Location="XS1_PORT_1K"  Name="I2S_DATA_IN"/>
+
+            <!-- Used for looping back clocks -->
+            <Port Location="XS1_PORT_1N"  Name="PORT_NOT_IN_PACKAGE_1"/>
```

Comment:
Are loopbacks needed for the app? I didn;t see mention in the readme

Reply:
Nope. Got copied from the FFVA example. Will delete.